test/integration/executor/command_permission_test.cpp

```diff
@@ -66,7 +66,7 @@ std::string makeDescription(ActorRolePermissions actor_role_permissions,
   static const EnumMap<ActorRolePermissions, std::string>
       kActorRolePermissionNames{
           {ActorRolePermissions::kNone, "no_role_permissions"},
-          {ActorRolePermissions::kMe, "role_permission_for_myself"},
+          {ActorRolePermissions::kMe, "role_permission_for_himself"},
```

Comment:
Not sure if I got the idea of this change

Reply:
this is changes the format of test case identifier like
```
Common/CallEnginePermissionTest.CommandPermissionTest/PostgreSQL___an_account_from_same_domain_having_role_permission_for_himself
```

I also changed the `me` case to be consistent, like
```
Common/CallEnginePermissionTest.CommandPermissionTest/PostgreSQL___same_account_having_role_permission_for_himself
```